multichain-testing/scripts/fund-provision-pool.ts

```diff
@@ -21,17 +24,37 @@ async function fundProvisionPool(args: {
   amount?: string;
   pod?: string;
   container?: string;
+  rpcUrl?: string;
 }) {
   const {
     address = DEFAULT_PROVISION_POOL_ADDR,
     amount = DEFAULT_PROVISION_POOL_COIN,
     pod = 'agoriclocal-genesis-0',
     container = 'validator',
+    rpcUrl = 'http://localhost:26657',
   } = args;
 
   console.log(`Funding provision pool at ${address} with ${amount}...`);
 
+  // not-DRY, but avoids setting up all of `e2e-tools.js`
```

Comment:
This should be available from client-utils. If it isn't please leave a TODO here

Reply:
I'm seeing a bunch of `wait*` functions, but nothing for this scenario:
```
'waitUntilAccountFunded',
'waitUntilContractDeployed',
'waitUntilElectionResult',
'waitUntilInvitationReceived',
'waitUntilOfferExited',
'waitUntilOfferResult',
```

Will add a TODO to implement in client-utils